.github/workflows/repo-milestone.yaml

```diff
@@ -0,0 +1,50 @@
+name: Repo / Auto-apply milestone
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - reopened
+
+jobs:
+  milestone:
+    name: Apply milestone
+    runs-on: ubuntu-22.04
+    if: ${{ contains(github.repository_owner, 'jellyfin') }}
+    steps:
+      - uses: actions/github-script@v6
+        id: my-script
```

Comment:
I believe the id can be removed if it's not used in the workflow

Reply:
Fixed.